digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -1818,6 +1818,53 @@ object DataFixture {
     }
   }
 
+  def addObstaclesShapefile(): Unit = {
+    println("\nStart process to add new obstacles by using the table created by the shapefile import")
+    println(DateTime.now())
+    println("")
+    val userProvider: UserProvider = new OracleUserProvider
+    val user = userProvider.getUser("k903846").get
+    val minimumDistanceFromRoadLink: Double = 3.0
+
+    println("\nGetting all obstacles information from the table created by the shapefile import")
+    val obstaclesInformation: Seq[ObstacleShapefile] = OracleDatabase.withDynSession {
+      obstacleService.getObstaclesFromShapefileTable()
+    }
+
+    OracleDatabase.withDynTransaction {
+      obstaclesInformation.foreach { obstacle =>
+        println("")
+        println("Creating a obstacle with coordinates -> " + "x:" + obstacle.lon + " y:" + obstacle.lat)
+        val pointObstacle = Point(obstacle.lon, obstacle.lat)
+
+        roadLinkService.getClosestRoadlinkFromVVH(user, pointObstacle, 10) match {
+          case Some(link) =>
+            val nearestRoadLinks = roadLinkService.enrichRoadLinksFromVVH(Seq(link))
+
+            if(nearestRoadLinks.nonEmpty){
+              val nearestRoadLink = nearestRoadLinks.head
```

Comment:
Doesn't have a rule in case of more than one nearest road link?
Does Mikko say something about that?

Reply:
It will only get one because when getting from VVH you only bring the closer one